### ISSUE TITLE
`RootFs` need to be cloneable, so the whole state can be copies instead of inherit a reference on `fork()`/`clone()`

### ISSUE THREAD

**Issue by michalfita** [nuta/kerla#105]
At the moment `Process::fork()` seem to clone the `Arc<RootFs>` instead of content of the `RootFs` itself, what leads to shared `RootFs` between processes, not independent file system states.

Found during work on #88 and belongs to `Multithreading` milestone (I can't add myself).

More details in [the flags mask of `clone(2)`](https://man.archlinux.org/man/clone.2#The_flags_mask), scroll down to `CLONE_FS`. I haven't checked yet how it's implemented in Linux Kernel.

**Comment by nuta** [comment]
I agree. We should make RootFs cloneable. Simply adding `#[derive(Clone)]` should work.

**Comment by nuta** [comment]
Just letting you know that according to clone(2)'s man page, mount points are controlled through another flag `CLONE_NEWNS` despite Kerla maintains them in `RootFs`. We'll need to isolate the `mount_points` field some day but for now, I think we don't need to support the flag unless we support running something like Docker.

**Comment by michalfita** [comment]
Yes, @nuta - all flags related to namespaces are for now excluded from my attempts, namespaces are huge topic related to contenerization.
